manager/integration/tests/common.py

```diff
@@ -6111,3 +6111,13 @@ def wait_for_instance_manager_count(client, number, retry_counts=120):
         time.sleep(RETRY_INTERVAL_LONG)
 
     return len(ims)
+
+
+def wait_delete_dm_device(api, name):
```

Comment:
Where is `api` used?

Reply:
Used by `create_and_verify_block_volume`